ios/RNSentry.mm

```diff
@@ -681,7 +682,7 @@ - (NSDictionary*) fetchNativeStackFramesBy: (NSArray<NSNumber*>*)instructionsAdd
         NSDictionary<NSString *, id> * nativeProfile = nil;
         if (nativeProfileTraceId != nil && nativeProfileStartTime != 0) {
 #if SENTRY_TARGET_PROFILING_SUPPORTED
-            uint64_t nativeProfileStopTime = [[[SentryDependencyContainer sharedInstance] dateProvider] systemTime];
+            uint64_t nativeProfileStopTime = clock_gettime_nsec_np(CLOCK_UPTIME_RAW);
```

Comment:
For future reference we can't use the `systemTime` from the shared container as it uses Swift and we can't use Swift code in the RNSentry C++ files, because the Native Module doesn't compile with Cpp modules.